package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name":     "jshint",
-  "version":  "2.4.4",
+  "name":     "fluid-jshint",
+  "version":  "2.4.5",
```

Comment:
Thinking if the infusion hint version of `2.4.5` would confuse with the actual jshint version with the same number. This also means `fluid-jshint` module would start from `2.4.5` without going thru earlier version numbers.